net/httpclient.go

```diff
@@ -133,6 +133,7 @@ func (c *Client) CloseIdleConnections() {
 // which can be nil to get the
 // https://godoc.org/github.com/opentracing/opentracing-go#NoopTracer.
 type Options struct {
+	Proxy func(req *http.Request) (*url.URL, error)
```

Comment:
`// Proxy see https://golang.org/pkg/net/http/#Transport.Proxy`

Reply:
So you mean that we drop the default nil case as written in the golang docs, but then everyone needs to do the inconvenient passing of the http.ProxyFromEnvironment (or how the name was). So for usability I would prefer it this way, because net.Client should be a better http Client configured with great defaults with a similar “interface”.

Reply:
Here I mean to add a comment like all other fields have to stay consistent.

Reply:
done, thanks!